business/apps.go

```diff
@@ -175,7 +175,7 @@ func (in *AppService) GetAppList(ctx context.Context, criteria AppCriteria) (mod
 			}
 		}
 		if criteria.IncludeHealth {
-			appItem.Health, err = in.businessLayer.Health.GetAppHealth(ctx, criteria.Namespace, appItem.Name, criteria.RateInterval, criteria.QueryTime)
+			appItem.Health, err = in.businessLayer.Health.GetAppHealth(criteria.Namespace, appItem.Name, criteria.RateInterval, criteria.QueryTime, valueApp)
```

Comment:
Not a problem but why do you drop `ctx` as the first parameter here, but keep it in, say, the analogous code in business/workloads.go?

Reply:
The trace was missing, but it's good to add them.
I'll update it.